test/SILGen/builtins.swift

```diff
@@ -577,13 +577,6 @@ func castBitPatternFromBridgeObject(_ bo: Builtin.BridgeObject) -> Builtin.Word
   return Builtin.castBitPatternFromBridgeObject(bo)
 }
 
-// SWIFT_ENABLE_TENSORFLOW
```

Comment:
I wonder, is it not possible to easily fix this test?

Reply:
@marcrasi confirmed that this test should be removed. Tests for `poundAssert` exist upstream.